From 173d1f3503969d90b66bb8c6895ad8fb536b2753 Mon Sep 17 00:00:00 2001
From: Akhil Goyal <akhil.goyal@nxp.com>
Date: Fri, 9 Jun 2017 02:22:22 +0530
Subject: [PATCH 033/767] arm64:dts: Add LS1043/46 USDPAA dts files in Makefile

Modification required for fsl,dpaa node placement.
Now the node is part of soc node.

Signed-off-by: Akhil Goyal <akhil.goyal@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile                   | 2 ++
 arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts | 4 +++-
 arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-usdpaa.dts | 4 +++-
 3 files changed, 8 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 755ee7e0add6..6a38ad985bd5 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -6,10 +6,12 @@ dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-qds.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-qds-sdk.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-rdb.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-rdb-sdk.dtb
+dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1043a-rdb-usdpaa.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-qds.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-qds-sdk.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-rdb.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-rdb-sdk.dtb
+dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-rdb-usdpaa.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1088a-qds.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1088a-rdb.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls2080a-qds.dtb
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts
index c279f30d1c8e..5940dc680227 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a-rdb-usdpaa.dts
@@ -10,7 +10,7 @@
 
 #include "fsl-ls1043a-rdb-sdk.dts"
 
-/ {
+&soc {
 	bp7: buffer-pool@7 {
 		compatible = "fsl,p4080-bpool", "fsl,bpool";
 		fsl,bpid = <7>;
@@ -92,6 +92,8 @@
 			fsl,fman-oh-port = <&fman0_oh2>;
 		};
 	};
+};
+/ {
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-usdpaa.dts b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-usdpaa.dts
index 593b87c5d726..953b20d2e76f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-usdpaa.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a-rdb-usdpaa.dts
@@ -10,7 +10,7 @@
 
 #include "fsl-ls1046a-rdb-sdk.dts"
 
-/ {
+&soc {
 	bp7: buffer-pool@7 {
 		compatible = "fsl,ls1046a-bpool", "fsl,bpool";
 		fsl,bpid = <7>;
@@ -85,6 +85,8 @@
 			fsl,fman-oh-port = <&fman0_oh2>;
 		};
 	};
+};
+/ {
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
-- 
2.17.0

